exp-player/addon/components/exp-video-preview.js

```diff
@@ -0,0 +1,105 @@
+    }.property('index'),
+
+    noPrev: function() {
+        return this.get('index') <= 0;
+    }.property('index'),
+
+    currentVideo: function() {
+        return this.get('videos')[this.get('index')];
+    }.property('index'),
+
+    didInsertElement() {
+    if (this.get('prompt') || !this.get('record')) return;
+      this.get('videoRecorder').start(`video-preview-${this.get('session.id')}`, this.$('#recorder'), {
+        record: true,
+        hidden: this.get('hideRecorder'),
+      });
+    },
+
+    actions:{
+        accept() {
+            this.set('prompt', false);
+            if (!this.get('record')) return;
+            this.get('videoRecorder').start(`video-preview-${this.get('session.id')}`, this.$('#recorder'), {
+                record: true,
+                hidden: this.get('hideRecorder'),
+            });
+        },
+        nextVideo() {
+            this.set('index', this.get('index') + 1);
+        },
+        previousVideo() {
+            this.set('index', this.get('index') - 1);
+        },
+    },
+
+    type: 'exp-video-preview',
+    meta: {
+        name: 'ExpVideoPreview',
+        description: 'TODO: a description of this frame goes here.',
+        parameters: {
+            type: 'object',
+            properties: {
+                index: {
+                    type: 'integer',
+                    default: 0
+                },
+                videos: {
+                    type: 'array',
+                    description: 'A list of videos to preview',
+                    items: {
+                        type: 'object',
+                        properties: {
+                            sources: {
+                                type: 'array',
+                                default: [],
+                                items: {
+                                    type: 'object',
+                                    properties: {
+                                        src: {type: 'string'},
+                                        type: {type: 'string'}
+                                    },
+                                    required: ['src', 'type']
+                                }
+                            }
+                        },
+                        required: ['sources']
+                    },
+                    default: []
+                },
+                record: {
+                    type: 'boolean',
+                    description: 'Whether or not to record the user while they view the previews',
+                    default: true
+                },
+                hideRecorder: {
+                    type: 'boolean',
+                    description: 'Hide the video recorder component',
+                    default: true,
+                },
+                prompt: {
+                    type: 'string',
+                    description: 'Require a button press before showing the videos',
+                    default: false
```

Comment:
I'm not totally clear on what this variable is for- can you clarify?

The type is specified as a string, but the default is a boolean. (we get away with this mainly because the schema issn't used to validate anything; it's currently informational only)

It's also used as both a string and booleany in the corresponding template:
Button label: https://github.com/CenterForOpenScience/exp-addons/pull/33/files#diff-e31824100dedf21c87504a4b3ce9a7bdR14

Conditional: https://github.com/CenterForOpenScience/exp-addons/pull/33/files#diff-e31824100dedf21c87504a4b3ce9a7bdR6